pdk/1.1/modules/ROOT/pages/policies-pdk-configure-features-timer.adoc

```diff
@@ -0,0 +1,161 @@
+
+[source,rust]
+----
+#[entrypoint]
+async fn configure(
+   launcher: Launcher,
+   Configuration(bytes): Configuration,
+   clock: Clock, // Inject the clock to be able to launch async tasks.
+   client: HttpClient,
+) -> Result<()> {
+
+
+   // set the period between ticks of the timer.
+   let timer = clock.period(Duration::from_secs(10));
+----
+
+After you inject the timer, share a reference of the `timer` with the `on_request` and `on_response` functions to begin creating delayed functions:
+
+[source,rust]
+----
+    launcher
+        .launch(on_request(|rs| request_filter(rs, &timer, &config)))
+        .await?;
+----
+
+You can now await timer functions in `on_request` and `on_response` functions, for example:
+
+[source,rust]
+----
+    let slept = timer.sleep(duration).await;
+    let tick = timer.next_tick().await;
+----
+
+NOTE: To view an example policy project that launches delayed functions, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/spike/README.md[Spike Policy Example].
+
+== Launch Asynchronous Tasks
+
+Use the timer to execute tasks independent of the request flow. To execute asynchronous tasks:
+
+. Define your task inside an `async` function.
+. In the policy configuration function call your function but don't await the returned future.
+. In the policy configuration function call the `launch` function with the `on_request` and `on_response` filter functions but don't await the returned future.
+. Use the `join!` macro to await both futures of your task function and `launch` function at the same time. 
++
+NOTE: To use the `join` macro, add the ‘futures’ crates as dependency in your `cargo.toml` file.
+
+Each worker executes a copy of the task. Configure a loop inside the function to execute periodic tasks. For example, the following code example awaits both functions:
+
+[source,rust]
+----
+let task = my_async_task(&timer, &config);
+let launched = launcher.launch(filter);
+let joined = join!(launched, task);
+----
+
+Add a loop to the `my_async_task` function creates periodic tasks, for example:
+
+[source,rust]
+----
+async fn my_async_task(timer: &Timer, config: &Config) {
+    // While the policy is still running.
+    // Wait for the next cycle.
+    while timer.next_tick().await {
+        // Execute periodic task
+    }
+}
+----
+
+To launch multiple asynchronous tasks, send a reference of the timer to your `async` functions and join them with the `launch` task, for example:
+
+[source,rust]
+----
+let task1 = my_async_task1(&timer, &config);
+let task2 = my_async_task2(&timer, &config);
+let launched = launcher.launch(filter);
+let joined = join!(launched, task1, task2);
+----
+
+NOTE: To view an example policy project that launches asynchronous periodic tasks, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/metrics/README.md[Metrics Policy Example^].
```

Comment:
No need to add the "^" anymore - antora takes care of it. OK to leave, but just fyi.